modules/statics/src/coins.ts

```diff
@@ -469,6 +476,7 @@ export const coins = CoinMap.fromCoins([
   erc20('pdata', 'Opiria Token', 18, '0x0db03b6cde0b2d427c64a04feafd825938368f1f', UnderlyingAsset.PDATA),
   erc20('pbtc', 'PegBitcoin Token', 8, '0x6a7041ff8cb4da0253a00bb1e548caf77c238bda', UnderlyingAsset.PBTC),
   erc20('peg', 'PegNet', 8, '0x996b396b88cc4a1d8df3dbd1c088cdfaee17e6d4', UnderlyingAsset.PEG),
+  erc20('penky', 'Penky', 18, '0x3a35e98cc7390a15d458bb85f9a12823e2ef25ed', UnderlyingAsset.PENKY),
```

Comment:
This does not exist on etherscan, are you sure this contract address is correct?

https://etherscan.io/address/0x3a35e98cc7390a15d458bb85f9a12823e2ef25ed

Reply:
i use this link: https://polygonscan.com/token/0x3a35e98cc7390a15d458bb85f9a12823e2ef25ed